src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Comment/CommentDetail.html

```diff
@@ -24,6 +24,10 @@
                         <i class="icon-ban-circle"></i>
                         {{ "TR__REPORT" | translate }}
                     </a>
+                    <a href="#" data-ng-click="delete()" data-ng-if="report">
```

Comment:
This should not be `ng-if="report"`, but a proper check whether we are allowed to delete.